Index: data/users.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from model.user import User\r\n\r\nstudent = User(\r\n    first_name='Leonid',\r\n    last_name='Chaliy',\r\n    email='leonid@example.com',\r\n    gender='Male',\r\n    phone_number='1234567890',\r\n    birth_day='28',\r\n    birth_month='August',\r\n    birth_year=2004,\r\n    subject='Maths',\r\n    hobby='Sports',\r\n    address='Moscow',\r\n    state='NCR',\r\n    city='Delhi',\r\n)
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/data/users.py b/data/users.py
--- a/data/users.py	(revision 640712ff66ef24166f9e5628eb4fb5f6b1198d62)
+++ b/data/users.py	(date 1774974922078)
@@ -11,6 +11,7 @@
     birth_year=2004,
     subject='Maths',
     hobby='Sports',
+    picture='avatar.png',
     address='Moscow',
     state='NCR',
     city='Delhi',
Index: tests/test_student_registration.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from data.users import student\r\nfrom pages.registration_page import RegistrationPage\r\n\r\nregistration_page = RegistrationPage()\r\n\r\n\r\ndef test_student_registration():\r\n    registration_page.open()\r\n    registration_page.register(student)\r\n    registration_page.should_have_registered(student)
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/tests/test_student_registration.py b/tests/test_student_registration.py
--- a/tests/test_student_registration.py	(revision 640712ff66ef24166f9e5628eb4fb5f6b1198d62)
+++ b/tests/test_student_registration.py	(date 1774974891113)
@@ -1,10 +1,36 @@
-from data.users import student
 from pages.registration_page import RegistrationPage
 
 registration_page = RegistrationPage()
 
 
 def test_student_registration():
-    registration_page.open()
-    registration_page.register(student)
-    registration_page.should_have_registered(student)
\ No newline at end of file
+    (
+        registration_page
+        .open()
+        .fill_first_name('Leonid')
+        .fill_last_name('Chaliy')
+        .fill_email('leonid@example.com')
+        .fill_gender('Male')
+        .fill_phone_number('1234567890')
+        .fill_birth_date('28', 'August', 2004)
+        .fill_subject('Maths')
+        .fill_hobby('Sports')
+        .fill_picture('avatar.png')
+        .fill_current_address('Moscow')
+        .fill_state('NCR')
+        .fill_city('Delhi')
+        .submit()
+    )
+
+    registration_page.should_have_registered(
+        ('Student Name', 'Leonid Chaliy'),
+        ('Student Email', 'leonid@example.com'),
+        ('Gender', 'Male'),
+        ('Mobile', '1234567890'),
+        ('Date of Birth', '28 August,2004'),
+        ('Subjects', 'Maths'),
+        ('Hobbies', 'Sports'),
+        ('Picture', 'avatar.png'),
+        ('Address', 'Moscow'),
+        ('State and City', 'NCR Delhi'),
+    )
\ No newline at end of file
Index: model/user.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from dataclasses import dataclass\r\n\r\n\r\n@dataclass\r\nclass User:\r\n    first_name: str\r\n    last_name: str\r\n    email: str\r\n    gender: str\r\n    phone_number: str\r\n    birth_day: str\r\n    birth_month: str\r\n    birth_year: int\r\n    subject: str\r\n    hobby: str\r\n    address: str\r\n    state: str\r\n    city: str
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/model/user.py b/model/user.py
--- a/model/user.py	(revision 640712ff66ef24166f9e5628eb4fb5f6b1198d62)
+++ b/model/user.py	(date 1774974900654)
@@ -13,6 +13,7 @@
     birth_year: int
     subject: str
     hobby: str
+    picture: str
     address: str
     state: str
     city: str
\ No newline at end of file
